### ISSUE TITLE
The main landing text needs to be improved

### ISSUE THREAD

**Issue by blokhin** [Materials-Consortia/materials-consortia.github.io#9]
In a private communication with an industrial provider, exploring the possibilities to support Optimade, I've received rather strong criticism:

- the current text is unclear and difficult to understand
- the value of supporting the Optimade is not immediately seen
- from a consumer point of view: a centralized access interface (or at least a library implementing it) is missing

As an example of good landing, the [Facebook API](https://developers.facebook.com/docs/apis-and-sdks) was given.

I think, it's very healthy criticism, let's have it in mind.

**Comment by ltalirz** [comment]
So far, the optimade.org web site has been mainly developer-centric.
With the paper being published, we should slowly transform it into a user-focussing web site;
PR https://github.com/Materials-Consortia/materials-consortia.github.io/pull/5 adds an interactive documentation of the API and so already takes a big step in this direction.

**Comment by rartino** [comment]
How is this for a brainstorming idea for the front page?:

The text could start with, rather then end with, defining what OPTIMADE is, i.e., 

------

The Open Databases Integration for Materials Design (OPTIMADE) is a [consortium of materials database providers](https://www.optimade.org/consortium) with the aim of  making materials databases interoperational.

The [OPTIMADE API](https://github.com/Materials-Consortia/OPTIMADE) is an industry standard API that unifies searching and retrieving information from materials databases. [→ More about the motivations behind OPTIMADE API](https://www.optimade.org/motivations)

-------

And then the rest of the front page could be just two dropdowns + a serch box with the text: *Try out searching materials databases using the OPTIMADE API". One dropdown with a list of "Aggreate (default)" + a choice of databases presently implementing OPTIMADE. A second dropdown with "Search for", `Structures`, `Calculations`. A request to a specific database can be submitted to that database endpoint; whereas the 'Aggregate' request can go to @blokhin's skyscanner (although, if we are "standardizing" on it to this degree, I would feel more comfortable if the development was made collaboratively under the Materials-Consortia organization.) There is also obviously a need for a linked (or fold-outable) help text about the filter.

Just a very open suggestion. What do you think?

**Comment by gmrigna** [comment]
Indeed, this page is meant for publicizing OPTIMADE. So far, it was indeed focused on the specifications (which was and is still important) but it would indeed be nice to illustrate what can be done with it. I thus very much like @rartino ideas...

**Comment by rartino** [comment]
I'll add another issue here: someone really should take a look at the meta tags we have for the front page, because Google is still showing the old OPTIMADE capitalization in the search results (and I cannot figure out why, but setting correct meta tags may solve it.)

**Comment by gmrigna** [comment]
The problem seems to be on Google side (with their cache)... I tried Bing, Yahoo, and DuckDuckGo. They all have the correct capitalization...

**Comment by ml-evs** [comment]
Is anyone interested in doing this? I see @blokhin self-assigned at the time (I will now remove the assignment to free it up).
